tools/accuracy_checker/accuracy_checker/adapters/classification.py

```diff
@@ -67,3 +70,15 @@ def process(self, raw, identifiers=None, frame_meta=None):
             result.append(single_prediction)
 
         return result
+
+    def _extract_predictions(self, outputs_list, is_multi_infer=False):
+        if not is_multi_infer:
+            return outputs_list[0] if not isinstance(outputs_list, dict) else outputs_list
+
+        output_keys = list(outputs_list[0].keys())
```

Comment:
if you use output_keys only as iterable object, you do not need conversion to list, dictionary keys also support iteration as well.